src/frontend/next/src/components/Posts/LoadAutoScroll.tsx

```diff
@@ -44,12 +44,13 @@ function LoadAutoScroll({ onScroll }: Scroll) {
     return () => {
       observer.unobserve(buttonRefCopy as HTMLButtonElement);
     };
-  }, [$buttonRef, onScroll]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
We should file a follow-up to figure out why our eslint doesn't like `[]` since I'd say it's a valid use case to not define deps, but want a one-time effect.